src/solvespace.h

```diff
@@ -76,8 +76,8 @@ typedef struct _cairo_surface cairo_surface_t;
     dbp("tri: (%.3f %.3f %.3f) (%.3f %.3f %.3f) (%.3f %.3f %.3f)", \
         CO((tri).a), CO((tri).b), CO((tri).c))
 
-#ifndef isnan
-#   define isnan(x) (((x) != (x)) || (x > 1e11) || (x < -1e11))
+#ifndef IsReasonable
+#   define IsReasonable(x) (((x) != (x)) || (x > 1e11) || (x < -1e11))
```

Comment:
The whole point of renaming would be to get rid of a name collision and all this preprocessor junk. This function should be `static inline` inside `namespace SolveSpace`, like the `WRAP` ones.

Reply:
Oh, ok.

Reply:
You want the WRAP one to be static too?

Reply:
Ah, I misremembered that it was already static. It makes zero difference either way.

Reply:
Done.  unless you want the 3 wrap functions inline too.